src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
We need a specific parameter by component using such metadata cache system (validator, serializer and now this one AFAIK) to avoid conflict. It's the responsibility of FrameworkBundle to provide correct performance for components in prod (with according config) and it should be easy to do (DX). Good performance is not an optional feature.

Btw it seems more logical to configure the cache system of core components in framework_bundle than in an optional unrelated bundle. IMO Doctrine Cache Bundle is for userland cache. Here we are speaking about caches (especially for Validator and Serializer) that are internals of core components and that make such components very slow when not enabled.

Btw there is some other problems that prevent using Doctrine Cache bundle right now:

* Doctrine Cache Bundle is not part of the standard edition, it must be installed separately (DX)
* Validator and Serializer already expose a related/similar config that cannot be removed until 4.0 (BC)
* It will not be coherent to add the cache configuration of PropertyAccess in Doctrine Cache as other similar definitions are in FrameworkBundle (again, see https://github.com/symfony/symfony-standard/blob/master/app/config/config_prod.yml#L4-L8)